cfme/scripting/quickstart.py

```diff
@@ -62,7 +62,7 @@
      " libffi-devel python-devel tesseract"
      " libpng-devel"
      " freetype-devel"),
-    ("Red Hat Enterprise Linux Server release 7", "nss",
+    ("Red Hat Enterprise Linux", "nss",
```

Comment:
If it doesn't match Workstation then let's add Workstation to the list, please, instead of removing the `release 7` postfix thus losing the version check.

Reply:
@jkrocil Maybe add a possibility of passing a `re.compile` and string and if `re.compile` is present, then it would try matching the regexp? What do you think?

Reply:
Either works for me, I'd let Alex decide. Personally though, I wouldn't bother...

Reply:
i jsut remembered tht there is a fun trick with python

startswith takes a tuple, so we could just put a tuple into the listings that match multiple versions